releases/NS30_GP/6.11/Makefile

```diff
@@ -0,0 +1,19 @@
+# Copyright (C) 2019 Richard Hughes <richard@hughsie.com>
+# Copyright (C) 2019 Mikhail f. Shiryaev <mr.felixoid@gmail.com>
+
+VENDOR=8Bitdo
+PROJECT_NAME=NS30_GP
+VERSION=6.11
+FIRMWARE_FILES=$(wildcard *.dat)
+METAINFO_FILES=ns30gp.metainfo.xml
```

Comment:
This should be sn30gp, not ns30gp. (The xml file needs to be renamed too.)

Reply:
For sure, thanks